AwesomeTSProject/App.tsx

```diff
@@ -16,7 +16,7 @@ import {SafeAreaView, useColorScheme} from 'react-native';
 import {Colors} from 'react-native/Libraries/NewAppScreen';
 
 import TopBar from './src/components/TopBar/Index';
-import CryptoList from './src/containers/CryptoList/CryptoList';
+import CryptoList from './src/containers/CryptoList/Index';
```

Comment:
Rename the Index to index.tsx, and Styles to styles.ts

And all the imports should be something like this: 

import CryptoList from './src/containers/CryptoList';

index is the default export of the folder